src/addon-manager.js

```diff
@@ -770,6 +771,13 @@ class AddonManager extends EventEmitter {
           console.log(`Killing ${packageName} plugin.`);
           pluginProcess.p.kill();
         }
+
+        // Remove devices owned by this add-on.
+        for (const deviceId of Object.keys(this.devices)) {
+          if (adapterNames.includes(this.devices[deviceId].adapter.id)) {
+            delete this.devices[deviceId];
```

Comment:
I don't think anything relies on the event, but this should probably call handleDeviceRemoved to perform the delete or emit a THING_REMOVED event.